dxtbx/format/FormatMultiImageJIT.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import, division
```

Comment:
*, print_function*
(you don't have to add this, it is enforced by a test anyway. But good habit.)